ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -65,6 +65,9 @@ public void requestInterstitialAd(@NonNull Context context,
       return;
     }
 
+    VungleInitializer.getInstance()
+            .updateCoppaStatus(mediationAdRequest.taggedForChildDirectedTreatment());
+
```

Comment:
what about com.google.ads.mediation.vungle.rtb.VungleRtbInterstitialAd ? don't we need same lines there? and in com.google.ads.mediation.vungle.rtb.VungleRtbBannerAd too